test-suite/src/test/java/io/micronaut/docs/function/client/aws/LocalFunctionInvokeSpec.java

```diff
@@ -80,10 +79,11 @@ interface MathClient {
 
         //tag::functionRnd[]
         @Named("round")
-        int rnd(float value);
```

Comment:
this change used to be necessary, don't think it is anymore. float and other primitive types can be written to text/plain still, for compat reasons.